examples/sales_analytics/.gitignore

```diff
@@ -0,0 +1,11 @@
+# Ballerina generates this directory during the compilation of a package.
```

Comment:
No need to maintain separate `.gitignore` file move the content to the root directory